### DISCUSSION TITLE
Where to set Reverse Object Relation Datatype?

### DISCUSSION THREAD

**Discussion by marrep** [pimcore/pimcore#9256] [Q&A / Support] [answered]
hi there,

I found this in the documentation but literally nowhere inside the pimcore admin area.
https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Object_Classes/Data_Types/Reverse_Object_Relation_Type.html

any hints?

cheers!

**Comment by weisswurstkanone** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/6142086/119848392-12272480-bf0c-11eb-8700-767c5de0c6dc.png)

**Reply by marrep** [reply]
sorry, I have forgot to say that I am trying to use this inside a field-collection

**Reply by weisswurstkanone** [reply]
@marrep , it should not be possible to add it to a field collection because it is simply not allowed/supported. Just on direct object contanairs. see https://github.com/pimcore/pimcore/blob/4b2c56033b914ce30d08b24b3d64279c8000e10a/bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/reverseObjectRelation.js#L24
